src/Http/BaseApplication.php

```diff
@@ -269,6 +269,11 @@ protected function buildContainer(): ContainerInterface
             $plugin->services($container);
         }
 
+        $event = $this->dispatchEvent('Application.buildContainer', ['container' => $container]);
```

Comment:
We should start adding constants for the event names. Today itself I had made a typo in an event name and has to waste a few precious seconds to realize my mistake and fix it :slightly_smiling_face:.